LiskHQ/lisk-sdk
# Fix white/fixed/blacklist config and manage it in peer pool for connections- Closes#4197#4213#4214

### What was the problem?

- Adding fixed/blacklist/whitelist peers in config file causes the node to crash
- Whitelisted peers should not be evicted through inbound eviction mechanism.
- We should not disconnect or evict with a fixed peer even during the shuffling.

### How did I solve it?

- Valid schema in config file for different lists
- Pass white/black/fixed list to peerpool to manage connections based on these lists

### How to manually test it?

Run the app and use the list to see if the particular behavior of a peer list works or not

### Review checklist

- [x] The PR resolves #4197 #4213 #4214
